data-loader/core/src/main/java/com/scalar/db/dataloader/core/dataimport/processor/ImportProcessor.java

```diff
@@ -340,6 +348,22 @@ private ImportTransactionBatchResult processTransactionBatch(
     return importTransactionBatchResult;
   }
 
+  /**
+   * Safely aborts the provided distributed transaction. If the transaction is null, this method
+   * takes no action. If an exception occurs during the abort operation, it is logged as an error.
+   *
+   * @param transaction the {@link DistributedTransaction} to be aborted, may be null
+   */
+  private void abortTransactionSafely(DistributedTransaction transaction) {
```

Comment:
Based on the comment:

```suggestion
  private void abortTransactionSafely(@Nullable DistributedTransaction transaction) {
```

But, it seems like this method is always called with non-null `transaction`? If so, maybe we don't need the null check.